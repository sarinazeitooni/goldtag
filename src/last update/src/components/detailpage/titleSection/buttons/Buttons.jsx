import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import {faClock, faShoppingBag, faShoppingBasket, faStar} from "@fortawesome/free-solid-svg-icons";
import React from "react";
function Buttons({data, titleSectionData}){
    return(
        <div>
            <button className='available-btn'> <FontAwesomeIcon icon={faShoppingBag} />{titleSectionData.available}</button>
            <div className="label-info-container">
                <div className="label-info">
                    <span className='info'><FontAwesomeIcon icon={faClock}/> {data.daynum} </span>
                    {titleSectionData.day}
                </div>
                <div className="label-info">
                    <span className='info'><FontAwesomeIcon icon={faShoppingBasket}/> {data.purchased} </span>
                    {titleSectionData.purchased}
                </div>
            </div>
            <div className='product-charity'>
                با خرید این محصول از گلدتگ <span style={{color:'#ff7a7a'}}>100 تومان</span> به <span style={{color:'#436fbe'}}>خیریه</span> کمک کرده اید.
            </div>
        </div>
    )
};
export default Buttons;