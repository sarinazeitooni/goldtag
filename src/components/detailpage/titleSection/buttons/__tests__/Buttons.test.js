import React from 'react';
import { shallow } from 'enzyme';
import Buttons from "../Buttons";
describe("Conditions component" , ()=>{
    it("it should render a div",()=>{
        const container = shallow(<Buttons data={''} titleSectionData={''}/>)
        expect(container).toMatchSnapshot();
    });
});