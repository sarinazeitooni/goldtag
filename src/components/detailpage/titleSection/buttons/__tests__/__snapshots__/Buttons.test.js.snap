// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Conditions component it should render a div 1`] = `ShallowWrapper {}`;
